synfig-studio/CMakeLists.txt

```diff
@@ -22,23 +21,6 @@ add_subdirectory(ui)
 file(MAKE_DIRECTORY "${SYNFIG_BUILD_ROOT}/share/appdata")
 file(MAKE_DIRECTORY "${SYNFIG_BUILD_ROOT}/share/applications")
 
-STUDIO_INTLTOOL_MERGE(
```

Comment:
The main idea is switch from Intltool to Gettext, not just remove it. As I can see here `org.synfig.SynfigStudio.appdata.xml` will not be generated, so no translations will be available.

Reply:
@ice0 Ohh.. got it . I will try to fix these ASAP

Thanks :)

Reply:
@ice0 Please tell me if I have to create a function like studio_intltool_merge or a valid replacement is already there. coz i can't find its replacement in the gettext.cmake file.

Thanks :)

Reply:
@ice0 Kindly help I am even trying to create my own STUDIO_MSGFMT_MERGE function, but i don't think it is the right way, there must be some alternative there in the msys/bin/msgfmt.exe but idk. Kindly Guide me further. 

Thanks :)

Reply:
Ok, I will take a look and comment here later.

Reply:
@ice0 Thanks You So Much. :)

Reply:
@ice0 Kindly guide me. So that i can work on this issue and i wanted to ask how do you use linux .. as i heard from others, it is better than windows for the purpose of development so how exactly do u use it ..i mean dual boot , virtual box  or its the only os in your system and what should i do in my windows pc??

Reply:
@ice0 And from what i noticed in the build log when i built it earlier, there was something like this,

-- Could NOT find Gettext (missing: GETTEXT_MSGMERGE_EXECUTABLE GETTEXT_MSGFMT_EXECUTABLE) (found version "0.19.8.1")
-- Found Intltool: C:/msys64/usr/bin/intltool-merge
-- Fixing intltool-merge command...
-- Found Gettext: C:/msys64/mingw64/bin/msgmerge.exe (found version "0.19.8.1")


so from this i guess, i have to include the MSGMERGEEXECUTABLE and GETTEXT MSGFMT EXECUTABLE as per the requirements as the alternative of the STUDIO_INTLTOOL_MERGE 

Kindly comment something about this also. Thanks :)